examples/loops/loops.md

```diff
@@ -0,0 +1,85 @@
+# Looping Constructs

```

Comment:
I think you should also update the `README` and add this page to the examples menu list.